designs-for-feedback/trigger-workflows-in-pipeline.md

```diff
@@ -62,11 +62,10 @@ unless it was invoked explicitly when the pipeline is triggered with:
 }
 ```
 
-The nested `condition` key is reused from job configuration as can be seen below
-for the `deploy` job, and actually accepts any boolean, not just pipeline
-parameters. It is needed to signify that this value is evaluated at
-configuration compilation time, as opposed to a possible future addition that
-could be evaluated at run time.
+The `when` key actually accepts any boolean, not just pipeline parameters,
+though pipeline parameters will be the only meaningful use of this feature until
+we implement others. Note that the nested `condition` key used in conditional
+jobs has been dropped, as there is no intrinsic need for it in this situation.
```

Comment:
~ `has been dropped for workflows`?

Reply:
I don't think we need to talk about it as dropped. We do need to think hard about rationalizing that `when` in this case is a sub-key of the workflow, whereas in the case of the `when` step command it is "above" the `steps` stanza, and on a step itself it's a subkey -- this inconsistency is not great, but perhaps tolerable. Should it be a different word, or do we live with it?

Reply:
I don't think it should be different word, because `when`/`unless` is _exactly_ the behaviour we're conveying here. I still dream about just getting the others in line, but them being ~inconsistent~ context-specific is fine by me otherwise.